olive/common/utils.py

```diff
@@ -327,3 +328,18 @@ def copy_dir(src_dir, dst_dir, ignore=None, **kwargs):
                 f"Received shutil.Error '{e}' but all required file names exist in destination directory. "
                 "Assuming all files were copied successfully and continuing."
             )
+
+
+def set_tempdir(tempdir: str = None):
+    """Set the root directory for tempfiles.
+
+    :param new_tempdir: new tempdir.
```

Comment:
Align the param name in comment with the `tempdir`.

Reply:
Oh I missed this. Thanks for the catch!
Updated